compiler/common-artifacts/exclude.lst

```diff
@@ -5,6 +5,8 @@
 
 #[[ optimize : Exclude from circle optimization(circle2circle) ]]
 ## TensorFlowLiteRecipes
+optimize(BidirectionalSequenceLSTM_000)
+optimize(BidirectionalSequenceLSTM_001)
```

Comment:
Could you add some comments about why you add  these recipes to `optimize` exclude list?

Reply:
@mhs4670go, I added these lines to exclude list, to ensure that the `./nncc test` works without errors. As an examples, if I do not add as above and test, then I get the following errors:
```
terminate called after throwing an instance of 'oops::UserExn'
  what():  Error: Not supported: BIDIRECTIONAL_SEQUENCE_LSTM
Aborted (core dumped)
compiler/common-artifacts/CMakeFiles/common_artifacts_deps.dir/build.make:3019: recipe for target 'compiler/common-artifacts/BidirectionalSequenceLSTM_000.opt.circle' failed
make[2]: *** [compiler/common-artifacts/BidirectionalSequenceLSTM_000.opt.circle] Error 134
CMakeFiles/Makefile2:18900: recipe for target 'compiler/common-artifacts/CMakeFiles/common_artifacts_deps.dir/all' failed
make[1]: *** [compiler/common-artifacts/CMakeFiles/common_artifacts_deps.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
```

The definition for `BIDIRECTIONAL_SEQUENCE_LSTM` comes later under `luci import` as I understand, so this is a temporary fix to avoid errors as mentioned above. Please let me know if there is a workaround that does not involve exclusion.

Reply:
How about adding _luci_ files first and then `TensorFlowLiteRecipes`?

Reply:
@seanshpark , thanks for the suggestion, I will mark this pull request as do-not-merge until all the luci files have been added and merged.